carcara/src/ast/printer.rs

```diff
@@ -360,6 +357,7 @@ impl fmt::Display for Constant {
                 }
             }
             Constant::String(s) => write!(f, "\"{}\"", escape_string(s)),
+            Constant::BitVec(val, width) => write!(f, "{}, {}", val, width), // TODO: comeback to this
```

Comment:
There are multiple valid ways of printing bitvector constants, but I think the simplest way is to use the `(_ bv...)` syntax:
```suggestion
            Constant::BitVec(val, width) => write!(f, "(_ bv{} {})", val, width),
```